app/views/draws/show.html.erb

```diff
@@ -12,5 +12,8 @@
     </tbody>
   </table>
 </div>
+<% if can_start_draw?(draw: @draw, policy: policy(@draw)) %>
```

Comment:
yea it's what I originally had and then I starting thinking about `@draw.status` and for some reason forgot that the policy could include that as well 😕.